aws/src/main/java/org/apache/iceberg/aws/AwsProperties.java

```diff
@@ -69,46 +69,46 @@ public class AwsProperties implements Serializable {
    *
    * <p>For more details: https://docs.aws.amazon.com/AmazonS3/latest/dev/serv-side-encryption.html
    */
-  public static final String S3FILEIO_SSE_TYPE = "s3.sse.type";
+  public static final String S3FILEIO_SSE_TYPE = S3FileIOProperties.S3FILEIO_SSE_TYPE;
```

Comment:
I think we can just mark these properties as deprecated? Not sure if we should remove it at 2.0 or 1.4 though, given this is a pretty big change, 2.0 is probably better?